app/src/main/java/com/ecchilon/sadpanda/imageviewer/ScreenSlidePagerAdapter.java

```diff
@@ -28,7 +30,9 @@ public Fragment getItem(int position) {
 		ScreenSlidePageFragment frag = new ScreenSlidePageFragment();
 		frag.setArguments(arguments);
 
-		loader.getImage(position + 1, frag);
+		loader.getImage(position)
```

Comment:
Arguably the loading should be done inside `ScreenSlidePageFragment`, though that might be outside the scope of this PR.